src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function getTextByLine(text:string, line: number) {
+    return text.split("\n").at(line);
+}
+
+export function isNumber(text: string) {
+    return !Number.isNaN(Number(text));
```

Comment:
```suggestion
    return !Number.isNaN(Number.parseInt(text, 10));
```

` ` (string with whitespace) char would be parsed to `0`.

Just finally checked implementation within vscode, this is incorrect way to do the parsing (have no idea why my linter didn't catch it).